CometServer/Server/GameServer.cpp

```diff
@@ -52,7 +52,7 @@ namespace server
 			backoff = 0s;
 			for (int i = 0; i < 10; i++)
 			{
-				if (protocol.Tick() < 0) backoff += def::max_socket_read_backoff / 10;
+				if (protocol.Tick() <= 0) backoff += def::max_socket_read_backoff / 10;
```

Comment:
I'm still unsure if the receive method can return 0, but even if it can, there's no way to parse 0 bytes into a meaningful packet, so let's just ignore that case.